inc/interfaces/jd_hw.h

```diff
@@ -23,6 +23,7 @@ void target_enable_irq(void);
 void target_disable_irq(void);
 void target_wait_us(uint32_t n);
 void target_reset(void);
+int target_in_irq(void);
```

Comment:
to check for re-entrancy?

Reply:
yes - the jd_send() apis cannot be called from IRQs